pylint_odoo/checkers/no_modules.py

```diff
@@ -232,6 +260,18 @@ def visit_callfunc(self, node):
                     self.add_message('translation-field',
                                      node=argument_aux)
 
+        # Check if cr.execute() has an sql-injection
+        if node.as_string().lower().startswith('cr.execute(') or \
```

Comment:
in the test_repo there are cases using:

`self.env.cr.execute('SELECT D...`

and
```python
cr = self._cr
cr.execute('SELECT DISTINCT child_id ...
```
do you mean when there is a `self._cr.execute('SELECT DISTINCT child_id ...'` ?